apps/core-app/src/legos/form.ts

```diff
@@ -543,4 +543,50 @@ export const COMMON_FORMS: Record<string, CustomFormLego> = {
     submitButtonText: 'Update Delegate',
     tx: TX.MANAGE_DELEGATE,
   },
+  RAGEQUIT: {
+    id: 'RAGEQUIT',
+    title: 'Ragequit',
+    subtitle: 'Members',
+    fields: [
+      {
+        id: 'tokenAmounts',
+        type: 'formSegment',
+        title: 'Step 1. Select voting and/or non-voting tokens to ragequit',
+        fields: [
+          {
+            id: 'sharesToBurn',
+            // @ts-expect-error: cannot resolve type within formSegment using custom fields sent from the core-app
+            type: 'ragequitToken',
+          },
+          // @ts-expect-error: cannot resolve type within formSegment using custom fields sent from the core-app
+          { id: 'lootToBurn', type: 'ragequitToken' },
+        ],
+      },
```

Comment:
so i beleive if you add these fields to https://github.com/HausDAO/daohaus-monorepo/blob/develop/libs/form-builder/src/components/CoreFieldLookup.ts#L19 you do not need the `@ts-expect-error`

Reply:
right but these are all fields in the form builder itself - not in the core app.